codecov.yaml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 79%
```

Comment:
I set the minimum coverage at 79% to make sure we don't drop below our current number for now. As we move to our target of 85% we can increase this.

Reply:
After getting the migrations excluded from the coverage count (since we can't test these) our coverage is just under 89% so I went ahead and bumped this up to 85%